pkg/apis/pipeline/v1beta1/swagger.json

```diff
@@ -1630,7 +1630,7 @@
           "type": "array",
           "items": {
             "default": {},
-            "$ref": "#/definitions/v1beta1.Param"
+            "$ref": "#/definitions/v1.Param"
```

Comment:
Thanks for the reviews @afrittoli  🙏 👍

These are the swap of the RefSource and Params in remote resolution API from V1beta1 to V1.

We had some discussion around this at https://github.com/tektoncd/pipeline/pull/6444#discussion_r1213317877. And it seems this won't be breaking  existent users of resovlers/remote resolutions.

Reply:
Thanks @JeromeJu - the bit that I'm missing is how that relates to the change of storage version.

Reply:
I think it is the mock resolution that's used in unit tests at https://github.com/tektoncd/pipeline/blob/8d1a90346918c71fa628db8d50fefb759079ddd1/test/resolution.go#L46. 

And it's a great point that maybe this should go into the commit message to be clear on the change?

Reply:
@Yongxuanzhang thanks. Sorry I think I'm missing something. What is the relationship between changing the CRD stored version and changes to the `v1beta1.ResolutionRequestSpec` to use `v1.Params`?

Reply:
Chatted with @afrittoli offline 🙏, seems that we had the concern of this PR being very large and touching too many things. Regarding the swaps of v1beta1 API fields(RefSource & Params) for remote resolution, currently exploring if it could be separated in another PR to break this current PR down so as to make merging this PR easier.

Reply:
> @JeromeJu we should add an action-required in the release notes - for anyone implementing their own custom resolver, they would have to update their code to use v1.Param and v1.RefSource

Sounds like this should be documented in https://github.com/tektoncd/pipeline/blob/main/docs/how-to-write-a-resolver.md?